data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/range_action/PstRange.java

```diff
@@ -28,6 +30,11 @@ public PstRange(@JsonProperty("networkElement") NetworkElement networkElement) {
 
     @Override
     public void apply(Network network, double setpoint) {
```

Comment:
shouldn't the `setpoint` be an int ?

We should add some javadoc to say whether it's a tap or an angle ... to avoid confusion later.

Reply:
As it comes in the ApplicableRange interface double remains less specific for the other ranges. If one day let's say we want to deal with angles, we can still use this interface.

But yes I will add some javadoc to specify that.